playwright/sync_api/_generated.py

```diff
@@ -683,7 +683,7 @@ def fulfill(
         json: typing.Optional[typing.Any] = None,
         path: typing.Optional[typing.Union[str, pathlib.Path]] = None,
         content_type: typing.Optional[str] = None,
-        response: typing.Optional["APIResponse"] = None
```

Comment:
I'll extract them into a separate PR tomorrow.

Reply:
Extracted into https://github.com/microsoft/playwright-python/pull/2646.

Reply:
Its hard to split the changes, since `black` [starts reading `[project.requires-python]` ](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#:~:text=By%20default%2C%20Black%20will%20infer%20target%20versions%20from%20the%20project%20metadata%20in%20pyproject.toml%2C%20specifically%20the%20%5Bproject.requires%2Dpython%5D%20field.%20If%20this%20does%20not%20yield%20conclusive%20results%2C%20Black%20will%20use%20per%2Dfile%20auto%2Ddetection.) from the `pyproject.toml`. Lets land together.